.github/workflows/continuous-integration.yaml

```diff
@@ -52,7 +60,7 @@ env:
   ACTION_RUN_URL: ${{ github.server_url }}/${{ github.repository }}/actions/runs/${{ github.run_id }}
   IS_PR: ${{ github.event_name == 'pull_request' }}
   IS_WEEKLY: ${{ github.event_name == 'schedule' || ((github.event_name == 'workflow_dispatch') && (github.event.inputs.event_name == 'weekly')) }}
-  IS_RELEASE: ${{ github.event_name == 'push' && startsWith(github.ref, 'refs/tags/') }}
+  IS_RELEASE: ${{ github.event_name == 'workflow_call' && github.event.inputs.event_name == 'release'}}
```

Comment:
now a release can only be triggered by a call from another workflow (with "release" as input)